src/locales/en-US.json

```diff
@@ -67,7 +67,7 @@
     "depositAmount": "Deposit amount",
     "deposits": "Deposits",
     "description": "Description",
-    "disclaimer": "By continuing you agree that Balancer Labs is not liable for any losses you may incur",
+    "disclaimer": "By continuing you agree that Symmetric or Cent Finance are not liable for any losses you may incur",
```

Comment:
Don't need to mention Cent Finance here, or do we?

Reply:
It may not be needed but regulators now consider open source put out by one party and used by another as leaving the first party with some liability. It's not clear to be whether this actually leaves any liability with Cent Finance so until there's clarity I will leave this catch all statement in place. I will check whether it's necessary though because it would be nicer to not include this reference.